lib/chef/provisioning/aws_driver/aws_resource.rb

```diff
@@ -5,6 +5,9 @@
 # Common AWS resource - contains metadata that all AWS resources will need
 module Chef::Provisioning::AWSDriver
 class AWSResource < Chef::Provisioning::AWSDriver::SuperLWRP
+  actions :create, :destroy, :nothing
```

Comment:
I just talked to him about it now, and I think I'm up to speed.  Nouns, not verbs!